src/components/Navigation/Navigation.jsx

```diff
@@ -82,9 +86,37 @@ const Navigation = () => {
   return (
     <div data-test="navigation-display">
       <React.Fragment key={anchor}>
-        <Button data-test="nav-button" onClick={toggleDrawer(anchor, true)}>
-          <MenuIcon />
-        </Button>
+        <Box sx={{ mb: 15 }}>
+          <AppBar
+            position="absolute"
+            data-test="app-bar-display"
+            sx={{ backgroundColor: "black" }}
+          >
+            <Toolbar>
+              <Grid container spacing={2}>
+                <Grid item xs={1}>
+                  <IconButton
+                    edge="start"
+                    color="inherit"
+                    aria-label="menu"
+                    data-test="nav-button"
+                    onClick={toggleDrawer(anchor, true)}
+                  >
+                    <MenuIcon />
+                  </IconButton>
+                </Grid>
+                <Grid item xs={9} />
+                <Grid item xs={2}>
+                  <img
+                    src="../../src/assets/SnapIT-logo.png"
+                    alt="logo"
+                    style={{ height: 40 }}
```

Comment:
⛏️  It looks like they want us to use `sx` now.
https://mui.com/system/the-sx-prop/

```suggestion
                    sx={{ height: 40 }}
```

Reply:
Addressed: commit d6259ad3db873d2a960329c601ca0120b6ff4ee6